test/extensions/filters/http/ext_authz/ext_authz_fuzz_test.cc

```diff
@@ -64,13 +65,26 @@ class FuzzerMocks {
         .WillByDefault(Return(OptRef<const Network::Connection>{connection_}));
     connection_.stream_info_.downstream_connection_info_provider_->setRemoteAddress(addr_);
     connection_.stream_info_.downstream_connection_info_provider_->setLocalAddress(addr_);
+
+    ON_CALL(decoder_callbacks_.stream_info_, dynamicMetadata())
+        .WillByDefault(Invoke([&]() -> envoy::config::core::v3::Metadata& {
+          // Must be set before this mock is called!
+          return *filter_metadata_;
```

Comment:
Consider changing `filter_metadata_` to a non-optional data member to avoid a dereference error.

Reply:
Done.